### PR TITLE
Fix broken validating packages scripts

### PR DESCRIPTION
**Pull request by JakeQuilty**
- Fixes out of date reference to docker container name
- Deletes out of date `/start`
- Moves `/install` to `/bin/install`
- Updates `README` to reflect changes

I am also wondering if we should put the `start`, `install`, and it's own `docker-compose.yml` in separate directory since it doesn't get used anywhere else

### CODE REVIEW

**Review by diverdane — DISMISSED** [review]
I'm not too familiar with these scripts, but the changes certainly look legitimate to me.

As for your comment about moving `start`, `install`, and `docker-compose.yaml` to their own directory, I don't know about the `start` and `install`, but I don't think `docker-compose.yaml` can be moved as it appears that the the scripts in `./bin` rely on that file being there.

File: install [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-container=$(docker ps -aqf "name=conjur-intro_conjur")
+container=$(docker ps -aqf "name=dap-intro_conjur")
```
Comment (jvanderhoof) [inline install:3]: If we reference the initial master setup by `bin/dap`:
```suggestion
container=$(docker ps -aqf "name=master-1.mycompany.local")
```
then we can run:
```sh
bin/dap --provision-master --version 5.11.0
```
to start a single DAP instance, then something like:
```sh
./install ~/Downloads/conjur-ui_2.10.9.1-e389f20_amd64.deb
```
to install a particular package.

**Comment by jvanderhoof** [comment]
@JakeQuilty, mind moving the `install` script into the `bin` directory?  It should have gotten moved, but I didn't realize anyone used the `install` script 😛 .  

It's likely beyond the scope of this PR, but we should get rid of the `start` script from the root (mental note on my part).

File: docker-compose.yml [hunk 2]
```diff
@@ -78,6 +78,17 @@ services:
       - ./system/configuration:/opt/cyberark/dap/configuration:Z
       - ./system/logs/master-3:/var/log/conjur:Z
       - ./files:/conjur_files
+
```
Comment (jvanderhoof) [inline docker-compose.yml:81]: With the name change suggested below, we shouldn't need to make this change.

**Comment by JakeQuilty** [comment]
@jvanderhoof if we move `install` to `bin` we are going to have to either move the `docker-compose.yml` file to `bin` too, or make a new docker-compose for the install script. Which route do you think is best?

**Comment by JakeQuilty** [comment]
@jvanderhoof I added your suggestions. I deleted the `/start` script, because I looked around and it wasn't being used anywhere else

**Comment by JakeQuilty** [comment]
Sorry I took so long to finish this. I was using a modified version of one of the scripts for a different issue I was working on, so I waited until I was finished with that to make these changes

**Review by jvanderhoof — APPROVED** [review]
Looks good!

**Pull request merged**